simphony/io/h5_cuds.py

```diff
@@ -66,7 +70,125 @@ def close(self):
         """
         self._handle.close()
 
-    def add_particles(self, particles):
+    def add_dataset(self, container):
+        """Add a CUDS container
+
+        Parameters
+        ----------
+        container : {ABCMesh, ABCParticles, ABCLattice}
+            The CUDS container to add to the engine.
```

Comment:
"The CUDS container to add to the engine." ->  "The CUDS container to be added to the file." or "The CUDS container to be added."